CHANGELOG.md

```diff
@@ -17,6 +17,12 @@ and includes an additional section for migration notes.
 
 ### Migration Notes
```

Comment:
I think this should be added under the v9 release migration notes. 
https://github.com/nasa/cumulus-orca/blob/develop/CHANGELOG.md#900-2023-10-05